scripts/countMerge.js

```diff
@@ -0,0 +1,29 @@
+const fs = require("fs");
+const db = require("../models").sequelize;
+const userModel = db.models.User;
+const userFileLocation = "users";
+
+const mergeUsers = async () => {
+  let users = fs.readdirSync(userFileLocation);
+  users = users.filter((e) => e.match(/.*\.json/gi));
+  for (const user of users) {
+    const filePath = `${userFileLocation}/${user}`;
+    const data = fs.readFileSync(filePath);
+    const count = JSON.parse(data);
+    const userInstance = await userModel
+      .create({
+        discordId: user.split(".")[0],
+        count: Object.values(count),
```

Comment:
A little weird, `JSON.parse(data);` should yield `{ count: x }` so you should be able to call `count.count` instead of `Object.values(count)`. Doesn't `Object.values()` return an array anyways?

Reply:
While `Object.values()` does return an array, both methods work. Changed to `count.count`. Also removed the unneeded `.then()`.